doc/doc-requirements.txt

```diff
@@ -1,6 +1,5 @@
 # documentation specific extra packages
-sphinx>=1.7
```

Comment:
can we keep sphinx as a dependency here? I use `doc/doc-requirements.txt` to install things in my dev environment so having this is nice because it brings in sphinx and remembers for me that it has to be >=1.7